src/components/feedback/style/feedback.less

```diff
@@ -15,6 +15,10 @@
   .progress {
     margin-top: 8px;
   }
+  textarea {
+    max-width: 320px;
+    max-height: 75px;
+  }
```

Comment:
@oterral I added the height because without we can have situation like this:

![screenshot from 2015-12-09 14-28-50](https://cloud.githubusercontent.com/assets/6643536/11686934/23b0aaf4-9e84-11e5-9d8d-e2d414ba5613.png)

It can also be fixed with an overlay but I found the `max-height` solution to be more pleasant.

OK to use `max-width:100%;`